index.html

```diff
@@ -188,7 +213,7 @@ <h3><img src="image/bullets/Rectangle_57bullet.svg" alt="O">Skills</h3>
                 </div>
             </div>
         </section>
-        <section>
+        <section class="contact-section" id="contact">
             <div class="contact">
                 <h3>I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</h3>
```

Comment:
- [ ] As the previous reviewer suggested, Please align the `Get in touch` button according to the Figma design. 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![f1](https://user-images.githubusercontent.com/70483835/165934582-83065fe9-a9bd-4404-a0c6-1c211cf23cae.png) | ![f2](https://user-images.githubusercontent.com/70483835/165934556-ffd1863e-7647-40ca-93d6-344de5e05a24.png)